CMakeLists.txt

```diff
@@ -210,6 +210,17 @@ add_library(ypspur SHARED
   src/yprintf.c
   src/utility.c
 )
+
+add_library(ypspur-comm SHARED
+  src/communication.c
+  src/serial.c
+  src/utility.c
+)
+
+target_link_libraries(ypspur-comm
+ ypspur
```

Comment:
ypspur is IPC library connecting user program to ypspur-coordinator.
it's not used in ypspur-comm.